vars/pipelineVars.groovy

```diff
@@ -50,4 +50,9 @@ class pipelineVars implements Serializable {
     String quayBaseUri = 'quay.io/cloudservices'
     String quayUser = "cloudservices+push"
     String quayPushCredentialsId = "quay-cloudservices-push-token"
+
+    String emailDefaultSentTo = 'mseri@redhat.com'
```

Comment:
This should be `insights-qa@redhat.com` I would think by default. Maybe @bsquizz has a better suggestion

Reply:
Sure that would be fine

Reply:
Removed pipelinevars. We can add later on need basis.